lib/fs.js

```diff
@@ -796,7 +796,7 @@ function readdirSync(path, options) {
   return options.withFileTypes ? getDirents(path, result) : result;
 }
 
-function fstat(fd, options, callback) {
+function fstat(fd, options = { bigint: false }, callback) {
```

Comment:
Why not just `options = {}`? (Same below.)

Reply:
I just copied fs/promises.js

https://github.com/nodejs/node/blob/427e5348a267b7c88879b47f8d942fc3e84b30e9/lib/internal/fs/promises.js#L352

Should I use `getOptions(options, {bigint: false})` to match the other functions?

Reply:
`getOptions()` does a bunch of other things that do not fit these APIs (like accepting a string as the encoding option), using default arguments should be fine considering we do this for many other APIs as well.